.github/workflows/build.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       RUSTC_WRAPPER: "sccache"
       CCACHE: sccache
       SCCACHE_GHA_ENABLED: "true"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04 # Needed for artifacts to work on older systems (due to glibc)
```

Comment:
I think servo is still using ubuntu-20.04 when creating its artifacts for publishing

Reply:
I though 22.04 was oldest image Github actions supports, apparently not, but it seems that clang is too old.